meta.md

```diff
@@ -0,0 +1,209 @@
+
+However, it is for a lack of wanting to be dictatorial that we aimed to apply PSR-2
+styling, rationale and stances (Described in Section 4, Approaches) in PSR-12 instead of
+establishing new conventions.
+
+The same approach is applied to the current PER.
+
+## 3. Scope
+
+### 3.1. Goals
+
+This PER shares the same goals as PSR-12.
+
+> The intent of this guide is to reduce cognitive friction when scanning code from
+> different authors. It does so by enumerating a shared set of rules and expectations
+> about how to format PHP code.
+> When various authors collaborate across multiple projects, it helps to have one set
+> of guidelines to be used among all those projects. Thus, the benefit of this guide is
+> not in the rules themselves, but in the sharing of those rules.
+
+This PER is an extension of PSR-12, and therefore also an extension of PSR-1. The basis of PER is
+PSR-12 and therefore a list of differences is provided below to assist with migration,
+but it should be considered as an independent specification.
+
+This PER will include coding style guidelines related to new functionality added to PHP
+after the publication of PSR-12. This PER will also include clarifications on the text of
+PSR-12.
+
+### 3.2. Non-Goals
+
+It is not the intention of this PER to add entirely new coding style guidelines. It will
+also not change anything stipulated in PSR-1 and PSR-12.
+
+## 4. Approaches
+
+The overarching approach is to attempt to apply existing PSR-12 styling and rationale to
+new functionality as opposed to establishing new conventions.
+
+### 4.1. Strict Types Declarations
+
+There was a discussion about whether strict types should be enforced in the standard
+https://github.com/cs-extended/fig-standards/issues/7. All were in agreement we should only
+use a MUST or MUST NOT statement and avoid the use of a SHOULD statement and nobody wanted
+to say that strict types could not be declared. The discussion was whether it should be
+considered a coding style item which should be covered or whether it was out of scope, and it
+was decided to be out of scope of a coding style guide.
+
+### 4.2. Finally and Return Types Declaration Spacing
+
+Numerous options were suggested, and they can be seen
+[here for return type declarations](https://gist.github.com/michaelcullum/c025f3870c9ea1dd2668#file-returntypesspacing-php) or
+[here for finally blocks](https://gist.github.com/michaelcullum/c025f3870c9ea1dd2668#file-finallyblocks-php)
+and the current implementation was chosen due to consistency with other parts of the standard.
+
+### 4.3. Enforcing short form for all type keywords
+
+PHP 7.0 introduced [scalar types declaration](http://php.net/manual/en/functions.arguments.php#functions.arguments.type-declaration)
+which does not support long type aliases. Therefore, it makes sense to enforce primary short type forms to be used to
+have uniform syntax and prevent possible confusion.
+
+### 4.4. Public Survey
```

Comment:
I would split this and other too-old-stuff coming from PSR-2 in a separate file... But maybe we can do it later, when we make changes to the PER.

Reply:
There's a lot of stuff from the meta doc we can probably just toss out at this point, and I'd honestly like to see the spec part split into multiple pages for ease of reading/editing.  That's all post-1.0 though.